databases/orientdb/src/main/java/com/gentics/mesh/graphdb/cluster/TopologyEventBridge.java

```diff
@@ -55,7 +54,6 @@ public TopologyEventBridge(MeshOptions options, Lazy<Vertx> vertx, Lazy<Bootstra
 		this.boot = boot;
 		this.manager = manager;
 		this.databaseStatusMap = hz.getMap(DB_STATUS_MAP_KEY);
```

Comment:
The map is queried for every transaction. How expensive is this operation?

Reply:
I don't know how expensive this is but I don't think we have any other option to share the topology state across the cluster. I also tried using a local map instead but this was not yielding stable results. I hope most of these changes can be removed in the future anyhow.